src/components/base-address/resources/default-schema.ts

```diff
@@ -1,5 +1,5 @@
-import { AddressValidationRules } from '@bcrs-shared-components/enums'
-import { SchemaIF } from '@/components/base-address/interfaces'
+import { AddressValidationRules } from '@/enums'
+import { SchemaIF } from '@/interfaces'
 import { baseRules, spaceRules } from '@/components/base-address/factories/validation-factory'
```

Comment:
I'm thinking this import should be local, as in `from "./factories/validation-factory"`. This is because the factories are internal and local to this component.

The reason I'm going on about this is that [`@/` maps to the source folder in THIS repo](https://github.com/bcgov/bcrs-shared-components/blob/4c62c41230422c5ceef744176fdc1bb04d6b5372/vite.config.ts#L15), but when these components are imported (as source files) in a UI, the UI's source folder won't have the same files. Therefore, these components should import from the "shared version" of enums or interfaces or whatever they need. Then these components should work correctly in this repo (ie, in Storybook and unit tests) and also in whatever UI they're used in.

cc: @leodube-aot

Reply:
This import should probably be `from '../factories/validation-factory'`. That resolved the error you are referencing Paul when I tested out your PR locally.

Reply:
Resolved the error and I ignored the other import errors. Storybook still runs now without any problems 👍 Thank you Leo.